### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by joren485**
This PR cleans up code (and files) from the [original repo (sentry-auth-github)](https://github.com/getsentry/sentry-auth-github/), that we do not use.  

This PR also fixes the pep8 violations.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,12 +1,2 @@
...
-
-[pytest]
-python_files = test*.py
-addopts = --tb=native -p no:doctest
-norecursedirs = bin dist docs htmlcov script hooks node_modules .* {args}
-
-[flake8]
-ignore = F999,E501,E128,E124,E402,W503,E731,C901
-max-line-length = 100
-exclude = .tox,.git,*/migrations/*,node_modules/*,docs/*
```
Comment (thomwiggers) [inline setup.cfg]: Is @se-bastiaan interested in adding lints or something? If so we shouldn't be deleting this I think.

Comment (joren485) [reply setup.cfg]: The `flake8` config is now kept (with some small changes). I also fixed all the pep8 violations.

Comment (se-bastiaan) [reply setup.cfg]: I copied basically everything from the original repo, this is one of them :p
However, that doesn't mean that I think lints are bad.

**Review by se-bastiaan — DISMISSED** [review]
The flake8 config is still missing, add that and 👍

**Comment by joren485** [comment]
> The flake8 config is still missing, add that and

Ah my bad. I forgot to add it.  
Fixed.

File: README.rst [hunk 2]
```diff
@@ -1,30 +1,24 @@
 GitHub Auth for Thalia
```
Comment (joren485) [inline README.rst]: This shouldn't say `GitHub`

**Review by thomwiggers — APPROVED** [review]

**Pull request merged**